demos/yolov4/Dockerfile

```diff
@@ -19,6 +19,6 @@ WORKDIR /demo
 COPY requirements.txt requirements.txt
 RUN pip install -r requirements.txt
 
-RUN pip install git+https://github.com/tryolabs/norfair.git@master
+RUN pip install git+https://github.com/tryolabs/norfair.git@master#egg=norfair[metrics]
```

Comment:
We actually don't need `metrics` in this demo. I added it because if not the demo wouldn't work. After a bit of debugging, I found that the missing package is `importlib-metadata` since this demo runs with python `3.7`. I will add it directly in the `requirements.txt` instead.